src/main/java/uk/gov/hmcts/juror/job/execution/service/SmtpServiceImpl.java

```diff
@@ -1,39 +0,0 @@
-package uk.gov.hmcts.juror.job.execution.service;
-
-import org.springframework.mail.SimpleMailMessage;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.JavaMailSenderImpl;
-import org.springframework.stereotype.Service;
-import uk.gov.hmcts.juror.job.execution.config.SmtpConfig;
-import uk.gov.hmcts.juror.job.execution.service.contracts.SmtpService;
-
-import java.util.Properties;
-
-@Service
-public class SmtpServiceImpl implements SmtpService {
-
-    public JavaMailSender getJavaMailSender(SmtpConfig config) {
-        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
-        mailSender.setHost(config.getHost());
```

Comment:
We will want to keep this class for future use

Reply:
Dead code should not be in live code. If the code is required in the future, it can be retrieved from source control.